resources/js/googleAnalytics.js

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
Is this needed?

Reply:
not really as the file name indicates what what the file contains, however for consistent guideline to add comment for each file